### DISCUSSION TITLE
CS2 POV demos support

### DISCUSSION THREAD

**Discussion by Xiguinilster** [akiver/cs-demo-manager#882] [Ideas]
Previously it allowed you to analyze POV files recorded with the console. I wanted to analyze some and there was no solution to analyze them.
I have already tried changing all the sources

**Comment by WangChuDi** [comment]
I encountered the same problem. Currently, it seems that entering the `record` command in the console can only record the POV perspective. If you want to see the demo, you need to first enter the corresponding map in the game (if it is a creative workshop map), and then read the demo through the `playdemo` command. But when playing the demo, you will find that you cannot switch the perspective. I guess this is the disadvantage of POV recording.

**Reply by Xiguinilster** [reply]
I wasn't referring to that, before you could extract the xls file from the same pov that I used to create the big data of my players from the practices since demos cannot be recorded in them
